vendors/vendor/boards/board/aws_tests/config_files/core_mqtt_agent_config.h

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
In this case I left the FIXME statements out intentionally. This file is only required for building. The exact values of any of these macros are irrelevant since the test project only tests the OTA PAL.

Reply:
Maybe add a note about this then?